html/browsers/browsing-the-web/navigating-across-documents/javascript-url-abort/javascript-url-abort-return-value-string.tentative.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<title>Aborting fetch for javascript:string navigation</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#navigate">
+<link rel="help" href="https://github.com/whatwg/html/issues/2590">
+<div id="log"></div>
+<iframe src="support/iframe-and-links.html"></iframe>
+<script>
+async_test(test => {
+  onload = () => {
+    const child = document.querySelector('iframe').contentWindow;
+    child.document.querySelector("#slowLink").click();
+    // The step below is in a timeout. The framed page can't communicate back mid-parse because that
+    // would involve running script, which makes that navigation "mature", and we need to do this
+    // before it matures.
```

Comment:
Isn't it already mature though if it has a tree you have access to?

Reply:
iframe-and-links.html is fully loaded, but the further nested iframe should be loading and not mature when the next link is followed.

Reply:
Ah right, because the first load takes longer than 100ms. Okay, this setup seems reasonable then.

Reply:
@zcorpan @annevk What guarantees that this timeout will fire before the navigation matures? The 1s timer on the trickle is in a different process (the server). Depending on OS scheduling, it can in fact end up replying before this timer fires.

Reply:
I think we rely on that assumption in a lot of tests. If we cannot rely on that, we basically cannot write these kind of tests...

Reply:
Sure you could.  For example, you could kick off two separate loads, to one of which the server responds after 100ms and to one of which it responds after 1000ms, and do your stuff when the 100ms load comes in....  That would ensure that you're using a single clock to time things.

Anyway, as written this test will randomly fail in testing infrastructure.